src/main/webapp/WEB-INF/web.xml

```diff
@@ -9,7 +9,7 @@
     <servlet-class>org.mitre.dsmiley.httpproxy.ProxyServlet</servlet-class>
     <init-param>
       <param-name>targetUri</param-name>
-      <param-value>http://localhost:8082/</param-value>
+      <param-value>http://localhost:8082</param-value>
```

Comment:
How can we change this to use different url ? Would that be necessary to integrate this with docker-compose stack?

Reply:
For this there is a separate file: `src/main/docker/web.xml`. The current file can be used for local testing.

Reply:
Right. I noticed the two files. But missed to notice the tiny detail. Nice